packages/headless/src/features/index.ts

```diff
@@ -69,6 +69,15 @@ export namespace ContextActions {
   export const removeContext = removeContextAlias;
 }
 
+import {
+  enableDebug as enableDebugAlias,
+  disableDebug as disableDebugAlias,
+} from './debug/debug-actions';
+export namespace Debug {
```

Comment:
It would be consistent with the other namespaces to end the name with `Actions`, so `DebugActions` in this case.


I saw a lot of great comments by the team on the PR, and I agree that dispatching an action is best for now. Controllers are meant to map 1-to-1 with some visual UI component. We can always create a controller in the future if we see it is needed.